rapid7_insightvm/plugin.spec.yaml

```diff
@@ -2182,6 +2182,18 @@ actions:
         description: 'Tag search criteria - options documentation: https://help.rapid7.com/insightvm/en-us/api/#section/Responses/SearchCriteria'
         type: object
         required: true
+      size:
```

Comment:
Set a default here so not all assets are returned

Reply:
@mhofert-r7 still need to address this